echopype/tests/visualize/test_plot.py

```diff
@@ -298,11 +303,13 @@ def test_water_level_Sv_dataset(water_level, expect_warning):
     Sv = echopype.calibrate.compute_Sv(echodata, **range_kwargs)
     ds = Sv.set_coords('echo_range')
     range_in_meter = ds.echo_range
-    single_array = range_in_meter.isel(frequency=0, ping_time=0).values
+
+    single_array = range_in_meter.sel(channel='GPT  18 kHz 009072058c8d 1-1 ES18-11',
+                                      ping_time='2017-07-19T21:13:47.984999936').values
 
     if isinstance(water_level, xr.DataArray):
-        if 'frequency' in water_level.dims:
-            original_array = single_array + water_level.isel(frequency=0).values
+        if 'channel' in water_level.dims:
+            original_array = single_array + water_level.isel(channel=0).values
```

Comment:
For consistency, it seems like the `isel(channel=0)` selection could be replaced with what's done in line 244, `sel(channel='GPT 18 kHz 009072058c8d 1-1 ES18-11')`

Reply:
Similar to my previous comment, we cannot do `sel` because of the test data. Here `channel='GPT 18 kHz 009072058c8d 1-1 ES18-11'` does not exist. This particular line also corresponds to the test data created on line 203.